modules/install/pages/upgrade-to-ipv6.adoc

```diff
@@ -0,0 +1,42 @@
+= Upgrading to IPv6
```

Comment:
This page is not about upgrading **to** IPv6 but rather about upgrading **of** IPv6 clusters to 6.5.